controllers/comments.controller.js

```diff
@@ -29,4 +30,19 @@ function postCommentByArticle(req, res, next) {
     });
 }
 
-module.exports = { getCommentsByArticle, postCommentByArticle };
+function deleteCommentById(req, res, next) {
+  eraseCommentById(req.params.comment_id)
+    .then((rowCount) => {
+      if (rowCount === 1) res.status(204).send();
+      else res.status(404).send({ msg: "Not found" });
```

Comment:
We don't want to send an error here - our custom error handlers should do this for us if we reject in the model